tee-worker/core/rpc-client/src/direct_client.rs

```diff
@@ -220,6 +236,20 @@ fn decode_from_rpc_response(json_rpc_response: &str) -> Result<String> {
 	}
 }
 
+fn decode_u32_from_rpc_response(json_rpc_response: &str) -> Result<u32> {
```

Comment:
Maybe we could merge it to `decode_from_rpc_response` but let's leave it to next time